src/main/java/com/evilcorp/bloom/service/CategoryService.java

```diff
@@ -0,0 +1,75 @@
+package com.evilcorp.bloom.service;
+
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.springframework.dao.DataIntegrityViolationException;
+
+import com.evilcorp.bloom.dto.CategoryDto;
+import com.evilcorp.bloom.model.Category;
+import com.evilcorp.bloom.repo.CategoryRepo;
+import com.evilcorp.bloom.exception.NestedSubcategoriesException;
+import com.evilcorp.bloom.exception.NotFoundException;
+
+public class CategoryService {
+
+  private final CategoryRepo categoryRepo;
+
+  public CategoryService(CategoryRepo categoryRepo) {
+    this.categoryRepo = categoryRepo;
+  }
+
+  public String capitalize(String categoryName) {
+    if (categoryName.isEmpty() || categoryName == null) {
+      return categoryName;
+    }
+
+    return Arrays.stream(categoryName
+        .split("\\s+"))
+        .map(word -> word.substring(0, 1).toUpperCase() + word.substring(1).toLowerCase())
+        .collect(Collectors.joining(" "));
```

Comment:
same to space here

Reply:
This is super nitpicking and it could be a little perfection in this specific case, so when interacting with others, choose well your battles sometimes it isn't worthy the time to debate with others what is better or not,

Reply:
done